### PR TITLE
feat(Templates): Add from Blank Workflow Card + Updated TemplateService

### PR DESCRIPTION
**Pull request by Elaina-Lee**
## Requirement Checklist

* [X] The commit message follows our guidelines

## Type of Change

* [ ] Bug fix
* [X] Feature
* [ ] Other

- Added blank workflow card for the template designer
- Added onBlankWorkflow as template service option, determining the behavior of the blank workflow click

## Impact of Change

* [X] **This is a breaking change.**
Adding a required function on Base TemplateService

## Screenshots or Videos (if applicable)

image

### CODE REVIEW

File: libs/designer/src/lib/ui/templates/cards/templateCard.tsx [hunk 1]
```diff
@@ -162,6 +169,52 @@ export const TemplateCard = ({ templateName }: TemplateCardProps) => {
...
+    }),
+    BLANK_WORKFLOW_DESCRIPTION: intl.formatMessage({
+      defaultMessage: 'Start with a blank workflow to build your integration process from scratch.',
+      description: 'Label text for the card that lets users start from a blank workflow',
+      id: 'nN1ezT',
+    }),
+  };
+
+  const onBlankWorkflowClick = () => {
+    TemplateService().onAddBlankWorkflow();
```
Comment (preetriti1) [inline libs/designer/src/lib/ui/templates/cards/templateCard.tsx:195]: Lets add some telemetry whenever this is getting clicked.

**Review by preetriti1 — CHANGES_REQUESTED** [review]

File: libs/designer/src/lib/ui/templates/cards/templateCard.tsx (hunk 1 above)
Comment (Elaina-Lee) [reply libs/designer/src/lib/ui/templates/cards/templateCard.tsx:197]: Added a telemetry aligning with the other templateCard

File: libs/designer/src/lib/ui/templates/cards/templateCard.tsx [hunk 2]
```diff
@@ -162,6 +169,54 @@ export const TemplateCard = ({ templateName }: TemplateCardProps) => {
...
+      description: 'Label text for the card that lets users start from a blank workflow',
+      id: 'nN1ezT',
+    }),
+  };
+
+  const onBlankWorkflowClick = () => {
+    LoggerService().log({
+      level: LogEntryLevel.Trace,
+      area: 'Templates.TemplateCard',
+      message: 'Create new workflow is selected',
```
Comment (preetriti1) [inline libs/designer/src/lib/ui/templates/cards/templateCard.tsx:194]: Blank workflow is selected

**Review by preetriti1 — APPROVED** [review]

File: libs/logic-apps-shared/src/designer-client-services/lib/base/template.ts [hunk 3]
```diff
@@ -12,4 +13,6 @@ export class BaseTemplateService implements ITemplateService {
   }
 
   public openBladeAfterCreate = (workflowName: string): void => this.options.openBladeAfterCreate(workflowName);
```
Comment (preetriti1) [inline libs/logic-apps-shared/src/designer-client-services/lib/base/template.ts:15]: Can you please make sure we send workflowName as undefined when it is multi workflow.

**Pull request merged**